lib/cosmos/api/limits_api.rb

```diff
@@ -54,16 +52,19 @@ module Api
     #   name and packet name
     def get_stale(with_limits_only = false, target_name = nil, scope: $cosmos_scope, token: $cosmos_token)
       authorize(permission: 'tlm', target_name: target_name, scope: scope, token: token)
+      stale_time = Time.now.sys.to_nsec_from_epoch - (30 * Time::NSEC_PER_SECOND)
```

Comment:
This probably needs to be a config setting somewhere

Reply:
Could be an argument to get_stale.  This method is going to take an eternity to run with a lot of packets....

Reply:
We don't call this regularly in any applications so I think that's ok for now. I'm going to make all arguments keyword args and add staleness_secs: 30.